src/ui/public/vis/agg_config.js

```diff
@@ -298,10 +299,17 @@ export function VisAggConfigProvider(Private) {
     }
 
     if (!this.type) return '';
-    let pre = (_.get(this.vis, 'params.mode') === 'percentage') ? 'Percentage of ' : '';
+    let pre = (this.isPercentage()) ? 'Percentage of ' : '';
```

Comment:
why not handle both the same ? (label and the value)
so you already have set the eventData.percent when you want your value to be rendered as percent,
so you only need to print different label in tooltip formatter when that happens (check is already there), no need to polute aggConfigs